index.md

```diff
@@ -3,15 +3,80 @@ layout: lesson
 root: .  # Is the only page that doesn't follow the pattern /:path/index.html
 permalink: index.html  # Is the only page that doesn't follow the pattern /:path/index.html
 ---
-FIXME: home page introduction
+# About the Lesson
+In this lesson, we will work on to focus on various accelerating modules that are implemented in LAMMPS for achieving a performance gain. First, we will discuss about the various options and then I'll show you an example to demonstrate how to decide on input parametrs to gain a performance boost. In the practical session, you will get a chance to work on another LAMMPS problem to test some of the accelearting tools that are embedded in LAMMPS. 
 
-<!-- this is an html comment -->
+The content of the lesson is as follows:
+
+1. What is benchmark?
+    * general definition
+    * real-world example of benchmarking: assessment of understanding
+    * Ask users to build a mental model to link between steps of benchmarking
+2. Why benchmarking is essential?
+<!--3. Benchmarking in HPC
+    * Software and hardware performance
+    * Factors affecting software performances
+    * Useful performance metrics in HPC
+        * walltime
+        * cpuh
+        * Flops
+        * Theoretical peak performance
+4. Speed-up
+    * Various ways to speed-up/accelarate a code
+    * parallel processing
```

Comment:
Parallelism is relevant because that is something they can control. The same with hardware utilisation (in particular GPU), you need to actively do something for it to work (this is different to GROMACS, where it automatically consumes whatever it finds).